src/components/date/__spec__.js

```diff
@@ -381,6 +385,10 @@ describe('Date', () => {
       it('does not open the date picker', () => {
         expect(instance.openDatePicker).not.toHaveBeenCalled();
       });
+
+      it('sets the input as disabled', () => {
+        expect(instance._input.disabled).toEqual(true);
```

Comment:
No an input has a native html attribute of `disabled`. Both props (`disabled` and `readOnly`) should effect this attribute.